man/man8/zpool-scrub.8

```diff
@@ -27,27 +26,21 @@
 .\" Copyright 2017 Nexenta Systems, Inc.
 .\" Copyright (c) 2017 Open-E, Inc. All Rights Reserved.
 .\"
-.Dd August 9, 2019
+.Dd May 27, 2021
 .Dt ZPOOL-SCRUB 8
 .Os
+.
 .Sh NAME
 .Nm zpool-scrub
-.Nd Begin a scrub or resume a paused scrub of a ZFS storage pool
+.Nd begin or resume scrub of ZFS storage pool
```

Comment:
nit: keep the "a" on at least the pool, if not the scrub too

Reply:
as above